src/auth/internal/utils.ts

```diff
@@ -24,34 +12,27 @@ export function inferEmailLoginFromURL(featheryClient: Client) {
   else return emailLoginFirebase(featheryClient);
 }
 
-export function inferAuthLogout() {
-  let logout;
+export async function inferAuthLogout() {
+  // logout may not have a value in certain cases, i.e. stytch is already logged out so there is no jwt
```

Comment:
i'd probably just get rid of this comment now, rather than rewrite it. the current conditional doesn't really need one